.gitignore

```diff
@@ -0,0 +1,23 @@
+*.iml
+build
+.gradle
+/local.properties
+.idea
+/.idea/caches/build_file_checksums.ser
```

Comment:
Do we need those?

Reply:
+1,
I'd recommend gitignoring the entire .idea directory